api-admin/src/main/resources/api/openapi.yml

```diff
@@ -721,24 +721,24 @@ paths:
           content: { application/json: { schema: { type: object } } }
         '500':
           $ref: '#/components/responses/ServerError'
-  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/notificationConfigs:
+  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/triggers:
     get:
-      summary: gets all the active notification configs for the environment
-      tags: [ notificationConfig ]
+      summary: gets all the active triggered actions for the environment
+      tags: [ trigger ]
       operationId: findByStudy
       parameters:
         - { name: portalShortcode, in: path, required: true, schema: { type: string } }
         - { name: studyShortcode, in: path, required: true, schema: { type: string } }
         - { name: envName, in: path, required: true, schema: { type: string } }
       responses:
         '200':
-          description: The notification configs
+          description: The triggered actions
           content: { application/json: { schema: { type: object } } }
         '500':
           $ref: '#/components/responses/ServerError'
     post:
-      summary: create a new notification config for the environment
-      tags: [ notificationConfig ]
+      summary: create a new triggered action for the environment
```

Comment:
Just curious - the wording "triggered action" makes me think it won't just be for notifications in the future. Is that true, or are triggers purely for notifications?

Reply:
yeah, in the far future, we'll want to support triggering things that aren't notifications.  but that's a ways off